test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -5078,6 +5078,104 @@ BOOST_AUTO_TEST_CASE(invalid_address_length)
 	CHECK_WARNING(text, "checksum");
 }
 
+BOOST_AUTO_TEST_CASE(interface)
+{
+	char const* text = R"(
+		interface I {
+		}
+	)";
+	success(text);
+}
+
+BOOST_AUTO_TEST_CASE(interface_constructor)
+{
+	char const* text = R"(
+		interface I {
+			function I();
+		}
+	)";
+	success(text);
```

Comment:
Problems with constructors:
- constructor is not part of the deployed bytecode, albeit the two things are mixed in the ABI
- which constructor to take in case of multiple inheritance?
- inheritance in Solidity is incapable of supporting it without major changes

We've decided to drop support for the constructor in the interface.